WalletWasabi.Gui/Tabs/WalletManager/LoadWalletViewModel.cs

```diff
@@ -624,6 +565,30 @@ public async Task<KeyManager> LoadKeyManagerAsync(bool requirePassword, bool isH
 			}
 		}
 
+		private async Task EnumerateHardwareWalletsAsync()
+		{
+			var cts = new CancellationTokenSource(TimeSpan.FromMinutes(3));
```

Comment:
it is not necessary but in other places you use an `using` statement.

Reply:
It's disposed. We already have a try-finally, so I thought, I just put the dispose to the finally instead of nesting the code with a using.

Reply:
We'll have usings statements with C# 8, so this conversation is soon to be obsolated.